.jenkinsci/bindings.groovy

```diff
@@ -29,30 +29,30 @@ def doJavaBindings(os, packageName, buildType=Release) {
       zip -r $artifactsPath *.dll *.lib *.manifest *.exp libirohajava.so \$(echo ${packageName} | cut -d '.' -f1); \
       popd"
   if (os == 'windows') {
-    sh "cp $artifactsPath /tmp/${env.GIT_COMMIT}/bindings-artifact"
+    sh "cp $artifactsPath /tmp/${GIT_COMMIT}/bindings-artifact"
   }
   else {
     sh "cp $artifactsPath /tmp/bindings-artifact"
   }
   return artifactsPath
 }
 
-def doPythonBindings(os, buildType=Release) {
+def doPythonBindings(PBversion, os, buildType="Release") {
   def currentPath = sh(script: "pwd", returnStdout: true).trim()
-  def commit = env.GIT_COMMIT
-  def supportPython2 = "OFF"
+  def commit = GIT_COMMIT
+  def supportPython2 = PBversion == "2" ? "ON" : "OFF"
+  def version = PBversion == "2" ? "python2" : "python3"
   def artifactsPath = sprintf('%1$s/python-bindings-%2$s-%3$s-%4$s-%5$s-%6$s.zip',
-    [currentPath, env.PBVersion, buildType, os, sh(script: 'date "+%Y%m%d"', returnStdout: true).trim(), commit.substring(0,6)])
+    [currentPath, version, buildType, os, sh(script: 'date "+%Y%m%d"', returnStdout: true).trim(), commit.substring(0,6)])
   def cmakeOptions = ""
   if (os == 'windows') {
-    sh "mkdir -p /tmp/${env.GIT_COMMIT}/bindings-artifact"
+    sh "mkdir -p /tmp/${GIT_COMMIT}/bindings-artifact"
     cmakeOptions = '-DCMAKE_TOOLCHAIN_FILE=/c/Users/Administrator/Downloads/vcpkg-master/vcpkg-master/scripts/buildsystems/vcpkg.cmake -G "NMake Makefiles"'
   }
   if (os == 'linux') {
     // do not use preinstalled libed25519
     sh "rm -rf /usr/local/include/ed25519*; unlink /usr/local/lib/libed25519.so; rm -f /usr/local/lib/libed25519.so.1.2.2"
   }
-  if (env.PBVersion == "python2") { supportPython2 = "ON" }
```

Comment:
Why did you remove this line while it is still in use on line 61? `cmake` command would fail since variable is not defined.

Reply:
I change it by this line:
https://github.com/hyperledger/libiroha/pull/5/files/57b1249e1293bc07f4a77e3b3155c53523568cf7#diff-13d8dd8a4496726a4cd57868d48d7040R43

`def supportPython2 = PBversion == "2" ? "ON" : "OFF"`